src/virtualenv/app_data/__init__.py

```diff
@@ -24,7 +24,7 @@ def __call__(self, parser, namespace, values, option_string=None):
     @staticmethod
     def _check_folder(folder):
         folder = os.path.abspath(folder)
-        if not os.path.exists(folder):
+        if not os.path.isdir(folder):
```

Comment:
should only check that it's a directory, not in general

there's also a TOCTOU bug here but it's unlikely to be hit